### PR TITLE
Add close title to popups

### PR DESCRIPTION
**Pull request by ponceta**
image

Small fix

### CODE REVIEW

**Comment by ponceta** [comment]
jenkins test

File: src/components/contextpopup/ContextPopupDirective.js [hunk 1]
```diff
@@ -6,12 +6,13 @@
```
Comment (gjn) [inline src/components/contextpopup/ContextPopupDirective.js]: Why do it in the contextpopup directive? Having it in the popup directive alone should be enough. No?

File: src/components/popup/PopupDirective.js [hunk 2]
```diff
@@ -149,6 +150,7 @@
             scope.titlePrint = $translate.instant('print_action');
             scope.titleHelp = $translate.instant('help_label');
             scope.titleReduce = $translate.instant('reduce_label');
+            scope.titleClose = $translate.instant('close');
```
Comment (gjn) [inline src/components/popup/PopupDirective.js]: Does not react to translation changes.

Comment (gjn) [inline src/components/popup/PopupDirective.js]: Please ignore both my comments. I was being stupid.

**Comment by oterral** [comment]
LGTM but I suggest in the future  we try to use th new version of angular-translate see https://github.com/geoadmin/mf-geoadmin3/issues/2310

**Pull request merged**